modules/rubiconBidAdapter.js

```diff
@@ -949,6 +882,63 @@ function addVideoParameters(data, bidRequest) {
   data.imp[0].video.h = size[1]
 }
 
+function applyFPD(fpd, mediaType, data) {
+  const map = {user: {banner: 'tg_v.', code: 'user'}, context: {banner: 'tg_i.', code: 'site'}, adserver: 'dfp_ad_unit_code'};
+  let obj = {};
+  let keywords = [];
+  const validate = function(e, t) {
```

Comment:
Can we name these vars something more useful?

The build will minify them, but for editing single letter vars are not ideal.

Reply:
Updated